shotover-proxy/examples/cassandra_bench.rs

```diff
@@ -3,14 +3,17 @@ use test_helpers::docker_compose::DockerCompose;
 use test_helpers::shotover_process::ShotoverProcess;
 
 /// e.g.
-/// cargo run --release --example cassandra_bench -- --config-dir example-configs/cassandra-passthrough -r 1000
+/// cargo run --release --example cassandra_bench -- --docker-compose example-configs/cassandra-passthrough/docker-compose-4.0.yml --topology example-configs/cassandra-passthrough/topology.yaml -r 1000
 /// or
-/// cargo run --release --example cassandra_bench -- --config-dir tests/test-configs/cassandra-passthrough-parse-request -r 1000
+/// cargo run --release --example cassandra_bench -- --docker-compose tests/test-configs/cassandra-passthrough-parse-request/docker-compose.yml --topology tests/test-configs/cassandra-passthrough-parse-request/topology.yaml -r 1000
 #[derive(Parser, Clone)]
 #[clap(name = "cassandra_bench", arg_required_else_help = true)]
 pub struct Args {
     #[clap(short, long)]
-    pub config_dir: String,
+    pub docker_compose: String,
```

Comment:
Do these changes relate to the rest of the PR? Can they be pulled into a separate PR?

Reply:
I needed to change it because of the different versions of the docker-compose added to `cassandra-passthrough`. I'll pull those changes out though.